src/baseCommands/user/permset/assign.ts

```diff
@@ -84,18 +84,42 @@ export abstract class UserPermSetAssignBaseCommand extends SfCommand<PermsetAssi
 
   private print(): void {
     if (this.failures.length > 0 && this.successes.length > 0) {
-      this.styledHeader('Partial Success');
       this.styledHeader('Permsets Assigned');
-      this.table(this.successes, { name: { header: 'Username' }, value: { header: 'Permission Set Assignment' } });
+      this.styledHeader('Partial Success');
```

Comment:
why did this styled header move below the `Permsets Assigned` one? Also, should the `Permsets Assigned` header be removed?